gcloud/pubsub/test_client.py

```diff
@@ -44,7 +81,7 @@ def test_list_topics_no_paging(self):
         self.assertEqual(next_page_token, None)
 
         self.assertEqual(len(conn._requested), 0)
```

Comment:
After the change, `conn._requested` has no entries because the methods in `Client`, `Topic`, and `Subscription` all use the API helpers, which are mocked out.  We could probably just assign `client.connection = None` everywhere for the same effect.